core/src/main/scala/org/apache/spark/serializer/KryoSerializer.scala

```diff
@@ -519,6 +520,7 @@ private[spark] class KryoSerializerInstance(
  * Interface implemented by clients to register their classes with Kryo when using Kryo
  * serialization.
  */
+@Stable
 trait KryoRegistrator {
```

Comment:
This is inconsistent with the PR title because this is `trait KryoRegistrator` instead of `KryoSerializer`, @robreeves . Did I misunderstand your intention?

Reply:
You're correct that it should be `KryoRegistrator`! I accidentally started to use the file name instead of trait name. I updated the references.